docs/dictionary/message/msgChanged.lcdoc

```diff
@@ -15,14 +14,25 @@ OS: mac, windows, linux, ios, android, html5
 Platforms: desktop, mobile
 
 Example:
-on msgChanged
-  set the text of field "Message" to msg
+on msgChanged pHandler, pLine
+  set the text of field "Message" of stack "Message Box" to msg
 end msgChanged
 
 Description:
 Use the msgChanged <message> to display or log the content of the
 <message box>.
 
-References: put (command), container (glossary), command (glossary),
-message (glossary), message box (glossary), property (glossary), 
-messageBoxLastObject (property), messageBoxRedirect (property)
+If the msgChanged <message> is not handled the content of the 
+<message box> will be logged to the system log unless the engine is
+running in no ui (command line) mode which will write the content to
+<stdout>.
+
+Parameters:
+handlerName (string): The name of the handler that modified the message
+box.
+
+lineNumber (integer): The line number within the script that modified
+the message box.
```

Comment:
oh... the parser doesn't check the result so it can be `msg | message <whatevs>`